python/src/opendp/extras/polars/__init__.py

```diff
@@ -421,7 +421,7 @@ def mean(
         │ ---    │
         │ f64    │
         ╞════════╡
-        │ ...    │
+        │ ...... │
```

Comment:
doesn't hurt anything, but `...` is sufficient. Doubled, this is essentially a regex like `.*.*`.

Reply:
I set float-precision to zero above to prevent random dataframe sizes. With this configuration, it can return `-inf`, which is four characters, leaving three spaces. But the doctest requires four spaces. So I doubled the `...` to keep things loosen the doctest requirement on spacing without breaking the table